site/viamguides/src/control-module/control-module.md

```diff
@@ -0,0 +1,577 @@
+     ![GitHub repository secrets](assets/repoSecrets.png)
+1. **Create a new release**: From the main code page of your GitHub repo, find **Releases** in the right side menu and click **Create a new release**.
+   ![GitHub create release](assets/createRelease.png)
+1. **Add a release tag**: In the **Choose a tag** dropdown, create a new tag such as `1.0.0`. Do not prepend the tag with `v` or the GitHub action will not trigger. For details about versioning, see [Module versioning](https://docs.viam.com/operate/reference/module-configuration/#module-versioning).
+   ![GitHub choose tag](assets/chooseTag.png)
+1. **Publish release**: Click **Publish release**. The cloud build action will begin building the new module version for each architecture listed in your `meta.json`.
+   ![GitHub publish release](assets/publishRelease.png)
+   > aside negative
+   > **Troubleshooting:**
+   >
+   > **GitHub action fails**: Check under the **Actions** tab in GitHub to view the status of the latest release. You can click on a failed workflow run to see more details. If the workflow fails and you need to retry the same version, you'll need to delete the tag from GitHub and re-push it to trigger the workflow again.
+   >
+   > **Update module**: If you make changes to the module code, remember to commit and push your updates to the remote GitHub repository, and then run `viam module update` from the command line.
+
+### Add your new modular resource to your machines
+
+Now that your module is in the registry, you can configure the component you added on your machines just as you would configure other components and services.
+
+1. **Delete local module configuration**: In the Viam app under your machine's **CONFIGURE** tab, delete the resource. There’s no more need for local module configuration since it is primarily for testing purposes. In our example, the local module configuration was called `controller`.
+1. **Delete local module**: For the same reasons, delete the local module. In our example, the local module was called `control-lamp-alarm`.
+   ![delete local module](assets/delete.png)
+1. **Add modular resource**: Click the **+** button, select "Component or service", and search for your model name to add your new modular resource.
+   ![add modular resource](assets/addModule.png)
+1. Configure the modular resource with the required attributes. Notice the beautiful documentation that you impressively added to your module, and the label in the top right corner of the resource panel indicating the module comes from the Viam registry.
+   ![new modular resource](assets/moduleAdded.png)
+1. **Save** your changes in the top right and wait a few moments for the configuration changes to take effect.
+
+   > aside negative
+   > **Troubleshooting:**
+   >
+   > If you're working within the same machine, you may need to **delete** the local module and local module configuration, instead of just disabling the resources.
```

Comment:
For the target device, i.e. Raspberry Pi, that was used for testing, you only need to disable hot reloading from the module card to get the published version from the registry. It will turn hot-reloading back on if you run the `viam module reload` command again with that part-id.

Reply:
Do you mean Disable hot reloading instead of Deleting the resources altogether? I tried disabling first, but my newly published module was reading from the local modules still (instead of registry).

Reply:
> Do you mean Disable hot reloading instead of Deleting the resources altogether? 

Yes, exactly. How do you know it was reading from the local module instead of the registry?

Reply:
The top right corner of the new element did not indicate it was coming from the registry, just showed the options (...) menu, and I checked the JSON view too.